spec/features/saml/aal3_required_spec.rb

```diff
@@ -15,11 +15,15 @@
     end
   end
 
-  describe 'SAML ServiceProvider configured to default to AAL3 authentication' do
-    context 'user does not have aal3 auth configured' do
-      it 'sends user to set up AAL3 auth' do
+  describe 'SAML ServiceProvider configured to default to phishing-resistant authentication' do
```

Comment:
**question:** should we rename this file?

Reply:
renamed in https://github.com/18F/identity-idp/pull/7063/commits/a1b5c69ebba565a3b6d35035082e0ef01fbae1cd